src/styles/footer.scss

```diff
@@ -0,0 +1,5 @@
+.footer {
```

Comment:
I haven't tried running it myself (`npm link` issues...), but I think these styles no longer apply now that you've removed the class and used the `<footer>` element?

Reply:
The class is added conditionally in https://github.com/solid/mashlib/pull/49/files#diff-f906bfae12622d8e92d22f8ed81de823R32

Reply:
Ah, understood.